src/Containers/Microsoft.NET.Build.Containers/PublicAPI/net472/PublicAPI.Unshipped.txt

```diff
@@ -47,8 +47,8 @@ Microsoft.NET.Build.Containers.Tasks.CreateNewImage.GeneratedContainerConfigurat
 Microsoft.NET.Build.Containers.Tasks.CreateNewImage.GeneratedContainerConfiguration.set -> void
 Microsoft.NET.Build.Containers.Tasks.CreateNewImage.GeneratedContainerManifest.get -> string!
 Microsoft.NET.Build.Containers.Tasks.CreateNewImage.GeneratedContainerManifest.set -> void
-Microsoft.NET.Build.Containers.Tasks.CreateNewImage.ImageName.get -> string!
-Microsoft.NET.Build.Containers.Tasks.CreateNewImage.ImageName.set -> void
+Microsoft.NET.Build.Containers.Tasks.CreateNewImage.Repository.get -> string!
+Microsoft.NET.Build.Containers.Tasks.CreateNewImage.Repository.set -> void
```

Comment:
`ContainerRepository`, just to stay consistent?
also do we need to change `BaseImageName`?